docs/run/nodes/nodes.md

```diff
@@ -266,12 +266,20 @@ The Ceramic daemon serves an HTTP API that clients use to interact with your Cer
 
 Ceramic nodes rely on IPFS for networking. IPFS nodes connect to each other using a Libp2p module called "switch" (aka "swarm"). This module operates over a websocket, on port `4011` by default. The websocket port must be accessible to the internet so your Ceramic node can be connected to the network.
 
+!!! warning ""
+
+    IPFS nodes connect to other nodes over the p2p network, and incoming connections to a node running on a local machine might get flagged as suspicious by a router or anti-virus software. Make sure to vet all such reports and verify on a case-by-case basis whether they are false positives and can be safely ignored.
```

Comment:
How would one go about this vetting?

Reply:
I'd look at the source IP address and port being accessed. My anti-virus tells me which ports are being accessed and from where. Are there any clarifications we can/should add here?